docs/docs/integrations/third_party_extensions.md

```diff
@@ -0,0 +1,17 @@
+---
+id: third_party_extensions
+title: Third Party Extensions
+---
+
+Being open source, others have the possibility to modify and extend the rich functionality Frigate already offers.
+This page is meant to be an overview over additions one can make to the home NVR setup. The list is not exhaustive and can be extended via PR to the Frigate docs.
+
+:::caution
+This page does not recommend or rate the presented projects.
+Please use your own knowledge to assess and vet them before you install anything on your system.
+
+:::
+
+## [Double Take](https://github.com/jakowenko/double-take)
+
+[Double Take](https://github.com/jakowenko/double-take) provides an unified UI and API for processing and training images for facial recognition.
```

Comment:
Might be worth noting here that doubletake supports automatically setting the sub labels for `Person` objects that are detected and recognized